### PR TITLE
DOCS-1578: Add data management and capture steps to Cartographer SLAM page

### PR DESCRIPTION
**Pull request by sguequierre**
* Update to reflect that data management and capture are no longer automatically configured in this workflow

### CODE REVIEW

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [inline docs/mobility/slam/cartographer/_index.md:106]:
```suggestion
   9. Select **SLAM**, then select `cartographer`.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [inline docs/mobility/slam/cartographer/_index.md:99]:
```suggestion
   6. Click `Add Method` and then select the method type and the capture frequency. For an RPlidar, choose the method [`NextPointCloud`](/components/camera/#getpointcloud).
```

File: docs/mobility/slam/cartographer/_index.md
Comment (EshaMaharishi) [inline docs/mobility/slam/cartographer/_index.md:101]: What do you think of moving the alert that says:

```
Be aware that data is not only captured when a slam session is running.
[Data Capture](/data/capture/) continuously monitors and captures your machine's sensor data while the machine is running.
To avoid incurring charges while not in use, [turn off data capture for your sensors](/data/capture/).
```

up to here, next to where we now talk about adding data capture?

File: docs/mobility/slam/cartographer/_index.md
Comment (EshaMaharishi) [inline docs/mobility/slam/cartographer/_index.md:134]: Should we make this same change for "Update existing map" below?

File: docs/mobility/slam/cartographer/_index.md
Comment (EshaMaharishi) [inline docs/mobility/slam/cartographer/_index.md:411]: Suggestion: "the capture rate" -> "data capture"

**Review by EshaMaharishi — COMMENTED** [review]
Thanks for your quick and neat work on this! Left a few minor comments

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:96]: Adding

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:99]: Adding, thanks!

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [inline docs/mobility/slam/cartographer/_index.md:411]:
```suggestion
The `cartographer` module on your machine polls the live LiDAR and IMU directly at these rates, whereas data capture is only used when data is being sent to the cloud.
```

Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:411]: Changed

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:101]: Sounds good, moved

File: docs/mobility/slam/cartographer/_index.md
Comment (EshaMaharishi) [inline docs/mobility/slam/cartographer/_index.md:102]: Thanks for moving this up here! This alert also currently appears under "Update an existing map", but I think we could remove it from there and just have it once here

**Review by EshaMaharishi — APPROVED** [review]
Looks good to me mod one final comment, but just a reminder we should not merge this until the corresponding frontend PR (https://github.com/viamrobotics/app/pull/3325) is merged. @kkufieta or I can comment on this PR once it's ready to be merged!

**Comment by sguequierre** [comment]
> Looks good to me mod one final comment, but just a reminder we should not merge this until the corresponding frontend PR ([viamrobotics/app#3325](https://github.com/viamrobotics/app/pull/3325)) is merged. @kkufieta or I can comment on this PR once it's ready to be merged!

Sounds good!

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:102]: I also found an alert in the "Localize Only" section and removed, is that ok or would you want to keep that one in? @EshaMaharishi

Comment (EshaMaharishi) [reply docs/mobility/slam/cartographer/_index.md:102]: Oh, good catch, not sure why that alert was there under "Localize Only" since data capture isn't even used there. Sounds good to remove!

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:1]: Small optional nit, really more of a question - do you want to include a link to the [Viam Rover](https://www.viam.com/resources/rover) on line 58?

Line 58:
>  For example, if you are using a Viam Rover and the [RPlidar A1](https://www.slamtec.com/en/Lidar/A1) model, mount it to the Rover so that the **pointed** end of the RPlidar mount housing is facing in the same direction as the webcam.

Suggestion:
> For example, if you are using a [Viam Rover](https://www.viam.com/resources/rover) and the [RPlidar A1](https://www.slamtec.com/en/Lidar/A1) model, mount it to the Rover so that the **pointed** end of the RPlidar mount housing is facing in the same direction as the webcam.

Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:1]: Tiny nit on word order, really subjective! I thought it might be nice if they read "only" before the end of the sentence.

Line 72:
>   Currently, the `rplidar` and `cartographer` modules support the Linux platform only.

Suggestion: 
>   Currently, the `rplidar` and `cartographer` modules  only support the Linux platform.

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:90]: Small optional nit for consistency in this list..! 
```suggestion
   1. Navigate to the **Services** subtab on your machine's **Config** tab.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:94]: I really love the detail in this. 

I thought possibly dividing it into two sentence might make it a bit easier to digest... what do you think though? Does this still retain your original meaning? Either way very subjective and optional. 

```suggestion
   4. On the panel that appears, you can manage the capturing and syncing functions. You can specify the **directory**, sync **interval**, and any **tags** to apply to captured data. See the [data management service](/data/) for more information.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:99]: Small optional nit! I thought changing "select" to "specify" would make the verb applicable to both "method type" and "capture frequency". 
```suggestion
   6. Click `Add Method`, then specify the method type and the capture frequency. For the camera, choose the [`NextPointCloud`](/components/camera/#getpointcloud) method. For a movement sensor, choose the [`AngularVelocity`](/components/movement-sensor/#getangularvelocity) and [`LinearAcceleration`](/components/movement-sensor/#getlinearacceleration) methods.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:103]: Super optional small nit. Do you think adding "data capture" could make it more clear which feature this is referencing/where in the app they would find it since it matches the name they'd see on the app?  Could also not be needed since the next sentence mentions it so explicitly!

```suggestion
   Be aware that data capture is not restricted to when a slam session is running.
```

File: docs/mobility/slam/cartographer/_index.md
Comment (skyleilani) [inline docs/mobility/slam/cartographer/_index.md:115]: small nit
```suggestion
   11. In the resulting `SLAM` service configuration pane, choose `Create new map` as the **Mapping mode**, then configure the rest of the **Attributes** for that mapping mode:
```

**Review by skyleilani — APPROVED** [review]
This looks great! I just left a few small and optional comments, feel free to ignore them. I couldn't comment directly on some of the lines, so I left some comments individually.

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:115]: nice

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:103]: Not implementing this just bc esha suggestion overwrites this sentence

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:99]: Done!

File: docs/mobility/slam/cartographer/_index.md
Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:1]: Done

Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:1]: Done! good idea

Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:1]: good w me~~

Comment (sguequierre) [reply docs/mobility/slam/cartographer/_index.md:1]: done!

File: docs/mobility/slam/cartographer/_index.md
Comment (npentrel) [inline docs/mobility/slam/cartographer/_index.md:81]: If captured here we need to also capture it on the matching json config tab

**Review by npentrel — CHANGES_REQUESTED** [review]

**Review by npentrel — APPROVED** [review]

**Pull request merged**
